benchmarks/scripts/expval_benchmark.py

```diff
@@ -78,26 +80,44 @@ def compile_for_simulation(
         case _:
             raise ValueError(f"Unknown compiler alias: {compiler_alias}")
 
-
-def simulate_density_matrix(circuit: qiskit.QuantumCircuit) -> np.ndarray:
+def depolarizing_error_model(one_q_err, two_q_err):
     depolarizing_noise = NoiseModel()
-    error = depolarizing_error(0.01, 1)
-    two_qubit_error = depolarizing_error(0.02, 2)
+    error = depolarizing_error(one_q_err, 1)
+    two_qubit_error = depolarizing_error(two_q_err, 2)
     # TODO: errors should only be added to the gateset that we are compiling to
     # but there is a bug with cirq currently compiling to U3 and CZ
-    depolarizing_noise.add_all_qubit_quantum_error(
-        error, ["u1", "u2", "u3", "rx", "ry", "rz", "h"]
-    )
-    depolarizing_noise.add_all_qubit_quantum_error(
-        two_qubit_error, ["cx", "cz"]
-    )
+    depolarizing_noise.add_all_qubit_quantum_error(error, ["u1", "u2", "u3", "rx", "ry", "rz", "h"])
+    depolarizing_noise.add_all_qubit_quantum_error(two_qubit_error, ["cx", "cz"])
+    return depolarizing_noise
 
-    simulator = AerSimulator(
-        method="density_matrix", noise_model=depolarizing_noise
-    )
+
+def simulate_density_matrix(circuit: qiskit.QuantumCircuit) -> np.ndarray:
+    simulator = AerSimulator(method="density_matrix", noise_model=depolarizing_error_model())
```

Comment:
Is this call to `depolarizing_error_model` missing the error rate arguments?

Reply:
Good catch, I changed where I set the default error rates and forgot to update `simulate_density_matrix`.